doc/backup_recovery.rst

```diff
@@ -0,0 +1,131 @@
+.. _backup_recovery:
+
+Recovery from backups
+=====================
+
+In case of emergency ... break glass.
+
+
+Backend
+-------
+
+Prepare a new RAID array
+........................
+
+In case of a real disaster, you will probably do the recovery from
+the real production instance. In case of a simulated disaster
+(i.e. testing the backups), spawn a new instance::
+
+    $ git clone git@github.com:fedora-copr/ansible-fedora-copr.git
+    # Follow the README.md steps for preparation
+    $ ./run-playbook on-own-risk-raid-10.yml
+
+Once the instance is spawned, see the instance details for its public
+IPv4 address, and SSH to the instance::
+
+    $ ssh fedora@xx.xxx.xxx.xxx
+    [fedora@ip-xx-xxx-xxx-xxx ~]$ sudo su
+    [root@ip-xx-xxx-xxx-xxx ~]#
+
+Set a root password, just in case we need to log in via EC2 Serial
+Console::
+
+    echo $RANDOM | md5sum | head -c 12; echo;
+    passwd
+
+Save the password in Bitwarden under the ``Temporary backup
+instances`` vault.
+
+Partition the disks::
+
+    for i in /dev/nvme[1-4]n1 ; do \
+        (echo gpt ; echo n ; echo ; echo ; echo ; echo ; echo w ) \
+        | sudo fdisk $i; done
+
+Create a new raid array::
+
+    mdadm --create /dev/md0 --level raid10 \
+        --name copr-backend-data --raid-disks 4 /dev/nvme[1-4]n1p1
+
+If the raid was successfully created, a check should be running by now::
+
+    cat /proc/mdstat
+
+You can see the raid details using::
+
+    mdadm --detail /dev/md0
+
+Format and mount::
+
+    mkfs.ext4 /dev/md0 -L copr-data
```

Comment:
I would add:
tune2fs -m0 /dev/md0
By default it is 5% and that is HUGE amount of data on large volumes.

Reply:
Updated